sklearn/linear_model/coordinate_descent.py

```diff
@@ -348,6 +348,9 @@ def enet_path(X, y, l1_ratio=0.5, eps=1e-3, n_alphas=100, alphas=None,
     ElasticNetCV
     """
     X = check_array(X, 'csc', dtype=np.float64, order='F', copy=copy_X)
+    if Xy is not None:
```

Comment:
Maybe @agramfort wants to have a look at this. This comes up in ElasticNetCV. In c2f0b31679a550b7c40cb733badc1e62ef0c7dbe I enforced it there, but there was a comment that is shouldn't be enforced there. not sure about that.